Neos.ContentRepository/Classes/Migration/Command/NodeCommandController.php

```diff
@@ -86,9 +86,9 @@ class NodeCommandController extends CommandController
      * @param boolean $confirmation Confirm application of this migration, only needed if the given migration contains any warnings.
      * @param string $direction The direction to work in, MigrationStatus::DIRECTION_UP or MigrationStatus::DIRECTION_DOWN
      * @return void
-     * @see neos.contentrepository.migration:node:migrationstatus
+     * @see neos.contentrepository.migration:node:migrate
      */
-    public function migrateCommand($version, $confirmation = false, $direction = MigrationStatus::DIRECTION_UP)
+    public function migrateCommand(string $version, bool $confirmation = false, string $direction = MigrationStatus::DIRECTION_UP): void
```

Comment:
These types are not really related to the documentation change.

Reply:
I agree. Will revert this change.